src/main/filterMod.F90

```diff
@@ -37,6 +37,9 @@ module filterMod
      integer, pointer :: soilnopcropp(:) ! soil w/o prog. crops (pfts)
      integer :: num_soilnopcropp         ! number of pfts in soil w/o prog crops
 
+     integer, pointer :: all_soil_patches(:) ! all soil or crop patches. Used for updating FATES SP drivers
```

Comment:
It looks like to me that this new filter is identical to the setFiltersOneGroup soilp filter, so you should be able to remove this one. We should make that an issue for bit-for-bit cleanup that could be done later.

Reply:
That's what I noticed too.  That said, there is something about the check on the activate columns or the `include_inactive` that changes science away from the expectation.  It could be that in trying to use the `soilp`  filter I wasn't appropriately understanding how the inactives are included.  I think maybe we could chat more with @rosiealice about this one for a future issue.

Reply:
Hmmm. OK. Should we file this as an issue to be looked at later? Or not worry about it?

Reply:
Per our chat, I've documented in #1488